data/projects.js

```diff
@@ -126,5 +126,26 @@ coala should support generating metrics for your code.",
 		"initiatives" : ["GSoC"],
 		"tags" : ["Core", "Bears"],
 		"markdown": "nested_languages.md"
+	},
+	{
+		"name" : "Improving coala website & supporting tools",
+		"desc" : "This project aims to intensify usage of coala by developing \
+		well organised web interfaces.",
+		"requirements" : [
+			"At least one patch to the coala core should be accepted and merged.",
+			"The student should be familiar with ongoing technical cEPs.",
+			"The proposal needs to contain a clear concept."
```

Comment:
probably something like some web dev stuff but I don't feel like this is a super complicated thing that needs deep requirements